### ISSUE TITLE
Nu-CRYO Idea : TARD-CBT

### ISSUE THREAD

**Issue by ShiftyRail** [d3athrow/vgstation13#25340]
TARD-CBT stands for **T**emperature **A**meliorated **R**eagent **D**istribution (for) **C**hems **B**ased **T**herapy

### Idea

Rather than outright removing so-called "duping", I would like to add a new machine to medical which would regulate the rate at which chems are distributed (and duplicated) inside a patient.

Lore-wise, it could be argued that duping consumes huge quantities of energy, and this energy must be dissipated in some way. The idea I had in mind was some sort of crystal that'd slowly heat up and be destroyed if you used cryo excessively and in unsafe ways (clothed patients, duping activated for too long...).

This crystal could be found by miners, or ordered from cargo, or any other way to obtain it. Personally I would like to get it from miners ; it gives miner a reason to interact with medical.

Duping more elaborate reagents - or using duping mode for an extended period of time - would of course cause the crystal to burn out faster.

### Why is this a good idea ?

Because I liked to push cryo to its limits in some cases. (after a bombing, after a shooting spree, etc, etc).
Because it would give medical a new machine to interact with (and argue about).
Because it would give traitors a new machine to sabotage. 
Because it would give miners a new reason to visit medical, and because it'd give mechanics a new machine to upgrade.

All of this is "WYCI", of course, and I can't give much time to coding right now, but I'll have more time to give in the following weeks, so here it goes as a "suggestion".

**Comment by boy2mantwicethefam** [comment]
tl;dr

**Comment by boy2mantwicethefam** [comment]
oh hey it wasn't that long
![bruh](https://i.imgur.com/DL6q2Qf.png)

**Comment by jknpj** [comment]
Just use supermatter splinters

**Comment by ShiftyRail** [comment]
Actually a good idea

**Comment by SonixApache** [comment]
did you really have to waste a precious issue number on this

**Comment by ShiftyRail** [comment]
Silly acronym aside, yes, I think it would be a good thing to add a new machine to medical.

**Comment by boy2mantwicethefam** [comment]
add the dialysis machine

**Comment by jknpj** [comment]
>Actually a good idea

I'll dub it as:

**R**adiation
**E**lectrolyzed
(and)
**T**emperature
**A**meliorated
**R**eagent
**D**istribution
(for)
**C**hems
**B**ased
**T**herapy

**Comment by boy2mantwicethefam** [comment]
actually that would be REATARDFCBT
